templates/offcanvas.mustache

```diff
@@ -0,0 +1,22 @@
+{{#regions.offcanvas.hasblocks}}
+    <div class="row">
+    {{#regions.offcanvasleft.hasblocks}}
+    <section class="theme-block-region block-region-offcanvasleft {{regions.offcanvas.class}}" id="offcanvasregionleft" aria-label="{{#str}}blocks{{/str}}">
```

Comment:
To help reading through the file, please indent the `<section>` tags within the 
`{{#regions.offcanvasleft.*}} `
sections